public/utils/constants.ts

```diff
@@ -18,3 +18,11 @@ export const BREADCRUMBS = Object.freeze({
   FLOW_FRAMEWORK: { text: 'Flow Framework' },
   WORKFLOWS: { text: 'Workflows', href: `#${APP_PATH.WORKFLOWS}` },
 });
+
+export const MDS_BREADCRUMBS = Object.freeze({
+  FLOW_FRAMEWORK: { text: 'Flow Framework' },
+  WORKFLOWS: (dataSourceId?: string) => ({
+    text: 'Workflows',
+    href: `#${APP_PATH.WORKFLOWS}?dataSourceId=${dataSourceId}`,
+  }),
+});
```

Comment:
I don't think we need `MDS_BREADCRUMBS` at all. We should just be able to dynamically handle the `dataSourceId` directly in the existing breadcrumbs and not have duplicate code. Something like:
```
export const BREADCRUMBS = Object.freeze({
  FLOW_FRAMEWORK: { text: 'Flow Framework' },
  WORKFLOWS: (dataSourceId?: string) => ({
    text: 'Workflows',
    href: constructHrefWithDataSourceId(APP_PATH.WORKFLOWS, dataSourceId, true),
  }),
});
```